package.json

```diff
@@ -28,6 +29,13 @@
   },
   "unpkg": "dist/js-yaml.min.js",
   "jsdelivr": "dist/js-yaml.min.js",
+  "exports": {
+    ".": {
+      "import": "./index.mjs",
+      "require": "./index.js",
+      "default": "./index.js"
+    }
+  },
```

Comment:
I suggest to also export `package.json`. Otherwise some tools might trip: https://github.com/nodejs/node/issues/33460